### PR TITLE
creat home page

### PR DESCRIPTION
**Pull request by Mariusz-web**

### CODE REVIEW

File: src/pages/Home/Home.scss [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    justify-content: center;
+
+    &__logo {
+        margin-bottom: 4rem;
+    }
+}
+
+@include for-phone-only {
+    .home__page {
+        &__logo {
```
Comment (Veryyapeee) [inline src/pages/Home/Home.scss:18]: shrink logo more, would be nicer I think

File: src/pages/Home/Home.jsx [hunk 2]
```diff
@@ -1,13 +1,12 @@
-import { Link } from 'react-router-dom';
-import { HelloWorld, Counter } from '../../components';
-import { routes } from '../../routes/Routes';
+import { ReactComponent as Logo } from '../../Images/logo.svg';
+import { Button } from '../../components/index';
+import './Home.scss';
 
 export const HomePage = () => {
```
Comment (Veryyapeee) [inline src/pages/Home/Home.jsx:5]: Add TODO Comment that we actually need to connect this button with google log in

**Review by Veryyapeee — CHANGES_REQUESTED** [review]

File: src/pages/Home/Home.scss (hunk 1 above)
Comment (Mariusz-web) [reply src/pages/Home/Home.scss:18]: width = 60% be better for u?

**Review by wikbro7650 — APPROVED** [review]

**Comment by Veryyapeee** [comment]
image

It's weird that the page is scrollable on mobile even if the whole content fits on the screen

**Review by PoProstuMieciek — APPROVED** [review]

**Review by Veryyapeee — APPROVED** [review]

**Pull request merged**